types/three/examples/jsm/exporters/EXRExporter.d.ts

```diff
@@ -1,7 +1,8 @@
 /**
  * @author sciecode / https://github.com/sciecode
  *
- * EXR format references: https://www.openexr.com/documentation/openexrfilelayout.pdf
+ * EXR format references:
+ * 	https://www.openexr.com/documentation/openexrfilelayout.pdf
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/three/examples/jsm/exporters/EXRExporter.d.ts;
  Line: 5;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.